Bugzilla/Template.pm

```diff
@@ -199,6 +199,13 @@ sub quoteUrls {
                ("\x{FDD2}" . ($count-1) . "\x{FDD3}")
               ~egox;
 
+    my $urlbase = Bugzilla->params->{'urlbase'};
+    $text =~ s~\bhttps?\Q://bugzil.la/\E([0-9]+)(\#c([0-9]+))?\b
```

Comment:
In the spirit of making this work, this is technically sufficient, and I have no objections. From a theoretical standpoint, copy-pasting the entire function just to avoid `(?: urlbase | bugzil.la )` really bugs me, because I don't know why at all they need to be separate other than that regexes are slightly annoying.

Reply:
the copy paste is because the second arg (the link description) should point to urlbase, not bugzil.la. This is to discourage people using bugzil.la